osidb/tests/test_flaw.py

```diff
@@ -73,7 +75,7 @@ def test_create(self, datetime_with_tz, good_cve_id):
             title="title",
             description="description",
             impact=FlawImpact.CRITICAL,
-            source=FlawSource.INTERNET,
```

Comment:
Because INTERNET is a public source and shouldn't have acknowledgments. If it has, an otherwise-unrelated validation fails. Hence the change to a private source.

Reply:
Then it should be in a different commit with some short description :)